dev/domain-controller/new-admin-vm/Install-CAM.ps1

```diff
@@ -1089,8 +1070,8 @@ graphURL=https\://graph.windows.net/
 
 				$catalinaHome = $using:CatalinaHomeLocation
 				$catalinaBase = "$catalinaHome\$using:brokerServiceName"
-				# I don't think we need to set the registry
-				# Set-ItemProperty -Path "$Reg" -Name CATALINA_BASE -Value $using:CatalinaHomeLocation
+
+                #set the current (temporary) environment
```

Comment:
CATALINA_BASE? That's no longer set in Install_Tomcat since the two instances have different CATALINA_BASE's